restapi/src/main/java/io/stargate/web/docsapi/service/query/ExpressionParser.java

```diff
@@ -177,6 +216,17 @@ private void validateFieldConditions(
               specificTypes.stream().map(Class::getSimpleName).collect(Collectors.joining(", ")));
       throw new ErrorCodeRuntimeException(ErrorCode.DOCS_API_SEARCH_FILTER_INVALID, msg);
     }
+
+    // If selectivity is explicitly provided, allow at most one condition to avoid ambiguity.
+    if (selectivity.isPresent() && conditions.size() > 1) {
```

Comment:
but multiple are anyway wrapped by and, I would gave them all same selectivity. WDYT? But maybe this is even better.. Let's keep it like this..

Reply:
Yes, but I'm not sure it is going to be obvious to the user :thinking: from my POV selectivity of an AND should be better than the selectivity of its sub-conditions, so using the same value there may be confusing.... I'd prefer to error out in that case and let the user define explicit values for each condition.